examples/tutorial/ArithmeticServer.hs

```diff
@@ -26,7 +26,7 @@ addHandler (ServerNormalRequest _metadata (TwoInts x y)) = do
 
 runningSumHandler :: ServerRequest 'ClientStreaming OneInt OneInt
                      -> IO (ServerResponse 'ClientStreaming OneInt)
-runningSumHandler req@(ServerReaderRequest metadata recv) =
+runningSumHandler _req@(ServerReaderRequest _metadata recv) =
```

Comment:
If `req` is unused, why keep the binding?

Reply:
Because typing `_` was easier, but now I've spent more keystrokes explaining it ;P. Will fix.